### PR TITLE
EIP-2: OS-specific commands to unzip in READMEs.

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket: https://issues.openmrs.org/browse/EIP-2

### CODE REVIEW

File: distribution/springboot_setup/README.md [hunk 1]
```diff
@@ -16,19 +16,27 @@ docker-compose up
...
 ```
 
 - a dump with lots of data for the remote database.
 ```
+#Command used on Linux
 cd distribution/springboot_setup/db
 zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
+
+#Command used on MacOS
+#unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
Comment (mks-d) [inline distribution/springboot_setup/README.md:39]: Note that the Apple Macintosh operating system is branded _macOS_ (with a non-capital m).
Let's format it like this and use it as a template for multi-OS commands documentation:

----

### 2. Import database dumps
Restore a database archive for each database:

##### a dump with not much data for the central database
<sub>Linux</sub>
```
cd distribution/springboot_setup/db
zcat dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```
cd distribution/springboot_setup/db
unzip -p dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```

##### a dump with lots of data for the remote database.
<sub>Linux</sub>
```
cd distribution/springboot_setup/db
zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```
cd distribution/springboot_setup/db
unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```

Comment (Ruhanga) [reply distribution/springboot_setup/README.md]: Cool. Thanks @mks-d.

File: distribution/springboot_setup/README.md [hunk 2]
```diff
@@ -16,20 +16,35 @@ docker-compose up
...
 ```
 cd distribution/springboot_setup/db
 zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
 ```
+<sub>macOS</sub>
+```
+cd distribution/springboot_setup/db
+unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
+```
+
```
Comment (mks-d) [inline distribution/springboot_setup/README.md:47]: @Ruhanga please replace the entire piece of Markdown that I provided under the line [here](https://github.com/openmrs/openmrs-eip/pull/8#discussion_r462112217).

Comment (Ruhanga) [reply distribution/springboot_setup/README.md:47]: Ohh, ok @mks-d. Could you share the raw markdown? I seemingly cannot extract it from your comment above, thanks.

Comment (mks-d) [reply distribution/springboot_setup/README.md:47]: @Ruhanga here:
`````
### 2. Import database dumps
Restore a database archive for each database:

##### a dump with not much data for the central database
<sub>Linux</sub>
```bash
cd distribution/springboot_setup/db
zcat dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```bash
cd distribution/springboot_setup/db
unzip -p dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```

##### a dump with lots of data for the remote database.
<sub>Linux</sub>
```bash
cd distribution/springboot_setup/db
zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```bash
cd distribution/springboot_setup/db
unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
`````

Comment (Ruhanga) [reply distribution/springboot_setup/README.md]: Thanks @mks-d. I've used the markup.

**Pull request merged**